package.json

```diff
@@ -1,13 +1,59 @@
 {
-  "devDependencies": {
-    "browser-sync": "^2.27.7",
-    "gulp": "^4.0.2",
-    "gulp-clean-css": "^4.3.0"
+  "name": "easy-popup",
+  "outputFilename": "easy-popup",
+  "prettyName": "Easy Popup",
+  "codeName": "EasyPopup",
+  "version": "0.0.6",
+  "description": "Super light-weight JavaScript library to create a simple popup with mobile layout.",
+  "homepage": "https://github.com/viivue/easy-popup",
+  "repository": {
+    "type": "git",
+    "url": "git@github.com:viivue/easy-popup"
+  },
+  "author": {
+    "name": "phucbm",
+    "url": "https://github.com/phucbm"
   },
-  "dependencies": {
-    "gulp-rename": "^2.0.0",
-    "gulp-replace": "^1.1.3",
-    "gulp-uglify-es": "^3.0.0",
-    "readline-sync": "^1.4.10"
+  "keywords": [
+    "phucbm",
+    "javascript",
+    "es6"
+  ],
+  "main": "./dist/easy-popup.module.js",
+  "license": "MIT",
+  "scripts": {
+    "dev": "webpack serve --config config/webpack.dev.js",
+    "build": "cross-env ENTRY=dev webpack --config config/webpack.build.js",
+    "sass:compressed": "sass src/_style.scss:dist/easy-select.css --no-source-map",
```

Comment:
`easy-select`?

Reply:
That's my fault. Sorry about that. I have changed it (more detail at https://github.com/viivue/easy-popup/pull/11/commits/ecaf7a369b834d48b432bfd23b5e893e196817a2), please check again! Thanks.